data/CollegeHero.sql

```diff
@@ -194,6 +194,35 @@ CREATE PROCEDURE getAllSectionInfoBySubject(IN subject VARCHAR(45))
     WHERE class.subject = subject;
   END//
 
+DROP PROCEDURE IF EXISTS getAllStudentOwingTuition;//
+CREATE PROCEDURE getAllStudentOwingTuiton(IN sID INT)
+    Begin
+        SELECT *
+        FROM student
+        GROUP BY sID HAVING tuition > 0;
+    END//
+
+DROP PROCEDURE IF EXISTS getAllStudentEnrolledInClass;//
+CREATE PROCEDURE getAllStudentEnrolledInClass(IN sID INT)
+    Begin
+        SELECT *
+        FROM student
+        WHERE student.sID = sID
+        UNION
+        SELECT *
+        FROM enroll
+        WHERE enroll.sID = sID;
+    END//
+
+DROP PROCEDURE IF EXISTS getAllClassInformationFromRoomNumber
+CREATE PROCEDURE getAllClassInformationFromRoomNumber(IN rID INT)
+SELECT a.cID,a.section,a.subject,a.tID,a.rID,a.days,a.start_at,a.end_at,a.capacity,a.cost
+FROM class a
+WHERE a.rID = (
+SELECT b.rID
+FROM room b 
+WHERE b.roomNumber = "001");
```

Comment:
This procedure is missing BEGIN and END tags, as well as not being formatted correctly.